htdocs/admin/tools/listevents.php

```diff
@@ -245,7 +245,7 @@
 	if ($limit > 0 && $limit != $conf->liste_limit) {
 		$param .= '&limit='.urlencode($limit);
 	}
-	if ($optioncss != '') {
+	if (!empty($optioncss)) {
```

Comment:
Same here. Following the template htdocs/modulebuilder/template/myobject_list.php
a better fix would be to initialize the $optioncss like it is into the example skeleton so adding at top of page
`$optioncss = GETPOST('optioncss', 'aZ'); // Option for the css output (always '' except when 'print')`